tasks/RedHat.yml

```diff
@@ -0,0 +1,20 @@
+---
+# RedHat family specific tasks for ansible-docker-installation
+- name: Add the Docker GPG key.
+  rpm_key: state=present key=https://yum.dockerproject.org/gpg
+
+- name: Add the Docker Repository.
+  yum_repository:
```

Comment:
In case you try to test this with a native packaged ansible this is a new module to ansible>=2.1.